### PR TITLE
[RHDEVDOCS-3157] Add broker ODC docs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
OCP 4.9+

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-3157

Link to docs preview:
http://file.rdu.redhat.com/abrennan/230622/RHDEVDOCS-3157/serverless/develop/serverless-using-brokers.html#serverless-creating-a-broker-odc_serverless-using-brokers

### CODE REVIEW

**Comment by mgencur** [comment]
LGTM

**Review by kelbrown20 — COMMENTED** [review]
Hiya! Just a small suggestion, feel free to discard if it messes with the context.

File: modules/serverless-creating-a-broker-odc.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+* You have logged in to the {product-title} web console.
+* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on the cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. In the *Developer* perspective, navigate to *+Add* -> *Broker*. The *Broker* page is displayed.
+
+. Optional. Update the *Name* of the broker. If you do not update the name, the broker that is created is named `default`.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply modules/serverless-creating-a-broker-odc.adoc:21]: I like it, thank you!

**Pull request merged**